comp/core/tagger/global.go

```diff
@@ -44,8 +44,25 @@ func GetEntity(entityID types.EntityID) (*types.Entity, error) {
 	return globalTagger.GetEntity(entityID)
 }
 
+// LegacyTag is an interface function that queries taggerclient singleton
+// If possible, avoid using this function, and use the Tag interface function instead.
+// This function exists in order not to break backward compatibility with rtloader and python
+// integrations using the tagger
+func LegacyTag(entity string, cardinality types.TagCardinality) ([]string, error) {
+	if globalTagger == nil {
+		return nil, fmt.Errorf("a global tagger must be set before calling Tag")
+	}
+
+	entityID, err := types.NewEntityIDFromString(entity)
+	if err != nil {
+		return []string{}, err
```

Comment:
Does it make a difference to return an empty slice instead of `nil` (like what is done above line 53) in case error ?

```suggestion
		return nil, err
```